src/logistration/RegistrationPage.jsx

```diff
@@ -249,9 +253,11 @@ class RegistrationPage extends React.Component {
             <ValidationFormGroup
               for="email"
               invalid={this.state.errors.email !== ''}
-              invalidMessage="Enter a valid email address that contains at least 3 characters."
+              invalidMessage={intl.formatMessage(messages['logistration.email.validation.message'])}
```

Comment:
Secondly a lot of changes are done in register form fields in another PR https://github.com/edx/frontend-app-logistration/pull/43 .Error messages are resolved realtime via backend and gets injected from state variables. So i don't think we can add translation like this. So error string would get translated via backend.